views/posts.ejs

```diff
@@ -0,0 +1,267 @@
+	        <a href="https://utopian.io/utopian-io/@<%= posts[index].get('author')%>/<%=posts[0].get('permlink') %>" class="post_title" title="<%= posts[index].get('title')%>" target="_blank"><%= posts[index].get('title')%></a> <!-- TODO: Add link to post -->
+				</div>
+	      <% }}); %>
+
+				<% if(posts.length<=0){%>
+ 				 <div class="empty-state">
+ 					 <br>
+ 					 <p class="h2">We don’t have anything<br> to show yet</p>
+ 					 <br>
+ 					 <p>Install <a href="https://chrome.google.com/webstore/detail/steemplus/mjbkjgcplmaneajhcbegoffkedeankaj" target="_blank" title="Install SteemPlus">SteemPlus chrome extension</a> and be among first.</p>
+ 				 </div>
+ 			 <% } %>
+
+      </div>
+
+
+			<footer id="footer" class="align-center">
+				<p>Created with <span class="heart">♥</span> for Steem and Utopian</p>
+				<p>Idea by <a href="https://steemit.com/@flauwy" target="_blank">@flauwy</a>, development by <a href="https://steemit.com/@stoodkev" target="_blank">@stoodkev</a>, design by <a href="https://steemit.com/@andrejcibik" target="_blank">@andrejcibik</a>.</p>
+      </footer>
+
+    </div>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+    <script src="https://cdn.steemjs.com/lib/latest/steem.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/parse/1.11.0/parse.min.js"></script>
+    <script>
+
+    $('.menu ul li a').eq(<%=active%>).addClass('active');
+
+    steem.config.set('websocket','wss://steemd.privex.io');
+
+
+    <%if(posts.length!==0&&(active==0||active==4)&&loggedIn&&account!==undefined&&posts.length!=0)
+    {%>
+      var reputation=steem.formatter.reputation(<%=account.reputation%>);
+      console.log(reputation);
+      if(reputation>=45)
+      {
+        console.log($('.1UP_votes').length);
+        $('.1UP_votes').css('cursor','pointer');
+        $('.1UP_votes').on('click', function(){
+          url="https://utopian.io/utopian-io/"+this.id;
+          console.log(url);
+          $.ajax({
+            type: "POST",
+            beforeSend: function(xhttp) {
+              xhttp.setRequestHeader("Content-type", "application/json");
+              xhttp.setRequestHeader("X-Parse-Application-Id", "efonwuhf7i2h4f72h3o8fho23fh7");
+            },
+            url: '/parse/classes/Votes',
+            data:JSON.stringify({"token":"<%=sToken%>","url":url}),
+            processData: false,
+            success: function(msg) {
+              console.log('voted!');
+              setTimeout(function(){
+              location.reload();
+              },750);
+            },
+            error: function(msg) {
+              alert(msg.responseJSON.error);
+            }
+          });
+        });
+      }
+    <%}%>
+
+      console.log("ACTIVE",<%=active%>);
+      if(<%=active%>==0)

```

Comment:
Whenever you do comparisons in javascript, please first try using `===` instead of `==`. 

e.g. because:
`1 == "1" // true`
`1 === "1" // false`